web-ide/ide-server/Dockerfile

```diff
@@ -14,16 +14,16 @@ RUN curl -sL https://deb.nodesource.com/setup_8.x | bash - &&\
 
 USER sdk
 ENV VERSION=0.11.19 \
-    CODESERVER_SHA256=355463f35b9c355cf99ea2f1d87367194daae8aa4fdabaace4a9ea2f1b0490f6
-RUN mkdir -p /home/sdk/.code-server/extensions/ &&\
+    CODESERVER_SHA256=4fe5b4d10d3048a5e5aa3e0772c09ece56241b91e76eeaa7404b4da292442881
+RUN mkdir -p /home/sdk/.local/share/code-server/extensions/ &&\
     mkdir /home/sdk/workspace &&\
-    curl -Lo - "https://github.com/codercom/code-server/releases/download/1.408-vsc1.32.0/code-server1.408-vsc1.32.0-linux-x64.tar.gz" | tar xzvf - --strip-components 1 "code-server1.408-vsc1.32.0-linux-x64" &&\
+    curl -Lo - "https://github.com/codercom/code-server/releases/download/1.939-vsc1.33.1/code-server1.939-vsc1.33.1-linux-x64.tar.gz" | tar xzvf - --strip-components 1 "code-server1.939-vsc1.33.1-linux-x64" &&\
```

Comment:
Can we use environment variables so these version numbers are only specified once?